.github/workflows/quality-checks.yml

```diff
@@ -9,15 +9,19 @@ jobs:
 
     strategy:
       matrix:
-        python-version:
-          - "3.12"
-          - "3.11"
-          - "3.10"
-          - "3.9"
+        python-version: ["3.12", "3.11", "3.10", "3.9"]
 
     steps:
       - uses: actions/checkout@v4
 
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
```

Comment:
@WillGibson install wise, this is where it happens ☝🏻

Reply:
I am very slow sometimes 😂